src/ucp/rndv/rndv_am.c

```diff
@@ -21,27 +21,40 @@ ucp_proto_rndv_am_init_common(ucp_proto_multi_init_params_t *params)
         return UCS_ERR_UNSUPPORTED;
     }
 
-    params->super.cfg_thresh =
-            ucp_proto_rndv_cfg_thresh(context, UCS_BIT(UCP_RNDV_MODE_AM));
+    params->super.min_length = 0,
+    params->super.max_length = SIZE_MAX,
     params->super.overhead   = 10e-9; /* for multiple lanes management */
     params->super.latency    = 0;
     params->first.lane_type  = UCP_LANE_TYPE_AM;
     params->middle.lane_type = UCP_LANE_TYPE_AM_BW;
     params->super.hdr_size   = sizeof(ucp_request_data_hdr_t);
     params->max_lanes        = context->config.ext.max_rndv_lanes;
```

Comment:
we are using `UCP_LANE_TYPE_AM_BW`, but referring to `context->config.ext.max_rndv_lanes` which manages `UCP_LANE_TYPE_RMA_BW`. Maybe use `context->config.ext.max_eager_lanes` instead?
(it's not related to the changes - just to discuss)